### PR TITLE
Update `Testing locally` section in `README.md`.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,14 +99,11 @@ parameters, in which case they do not specify `oauth2` or `zcap` properties.
...
 #### Testing locally
 
-If you want to test your implementation locally, you can add a configuration
-file in the root directory of the specific test suite that you are running.
-
-```
-localImplementationsConfig.cjs
-```
+If you want to test your implementations for endpoints running locally, you can
+create a configuration file `localImplementationsConfig.cjs` in the root
```
Comment (BigBlueHat) [inline README.md:103]: Does this need a `.` prefix? The text here and in the similar CCG repo https://github.com/w3c-ccg/vc-test-suite-implementations/pull/98 mention them both with and without the `.`, so we should clarify that both places--one way or the other.

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
We should also make sure `.gitignore` gets updated.

File: README.md (hunk 1 above)
Comment (JSAssassin) [reply README.md]: Thanks, fixed here [4057386](https://github.com/w3c/vc-test-suite-implementations/pull/10/commits/40573864081f9834fb293698b8c2600d4d1987ce).

**Review by BigBlueHat — APPROVED** [review]

**Comment by JSAssassin** [comment]
> We should also make sure `.gitignore` gets updated.

@BigBlueHat Do you mean we should add `.localImplementationsConfig.cjs` to `.gitignore` file in various test suites or are you suggesting we should update the `.gitignore` file here?

**Comment by BigBlueHat** [comment]
> @BigBlueHat Do you mean we should add `.localImplementationsConfig.cjs` to `.gitignore` file in various test suites or are you suggesting we should update the `.gitignore` file here?

Oof...yeah. It'd have to be loads of places... Let's leave it out for now, but we an add it if folks start to send PRs with it--or if we're 
doing other non-code updates to things like readmes, etc. Let's leave it alone for now. Thanks!

**Pull request merged**